### PR TITLE
Add an initial Dockerfile.

### PR DESCRIPTION
**Pull request by dakoner**
Hi Max,

It was a pain to get keras installed on my machine to run the examples in your repo, so I built this Dockerfile.  After docker build . I ran the container and could preprocess and started training (using CPUs).

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+RUN echo "force-unsafe-io" > /etc/dpkg/dpkg.cfg.d/02apt-speedup
+RUN echo "Acquire::http {No-Cache=True;};" > /etc/apt/apt.conf.d/no-cache
+
+RUN \
+  apt-get update && \
+  apt-get -y install \
+          software-properties-common
+
+RUN apt-get -y install python-pip git postgresql-server-dev-9.5 graphviz
```
Comment (maxhodak) [inline Dockerfile:18]: what's the reason for including `postgresql-server-dev-9.5` ? that's a pretty heavy dependency. (view towards chembl sql downloader?)

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+RUN pip install --upgrade pip
+
+COPY requirements.txt /tmp
+RUN pip install -r /tmp/requirements.txt
+RUN TF_BINARY_URL=https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-0.11.0rc1-cp27-none-linux_x86_64.whl && pip install --upgrade $TF_BINARY_URL
```
Comment (maxhodak) [inline Dockerfile:23]: I don't usually use docker so I'm not super familiar here, but it's too bad if it precludes GPUs.

File: Dockerfile (hunk 1 above)
Comment (dakoner) [reply Dockerfile:18]: requirements.txt listed psycopg2 so I assumed the code used postgres (and found that required to build psycopg2).  If you don't use that library, let me know, I will just remove psycopg2 from requirements.txt and  postgresql-server-dev-9.5 from Dockerfile.

File: Dockerfile (hunk 2 above)
Comment (dakoner) [reply Dockerfile:23]: I have a subsequent change that addresses this via command-line argument.

File: Dockerfile (hunk 1 above)
Comment (maxhodak) [reply Dockerfile:18]: My mistake - I was using psycopg2 elsewhere in a codebase I extracted this repo from where I was working with ChEMBL in Postgres and I copied over the requirements.txt. But it's not a dependency here. Go ahead and remove it, thanks!

**Pull request closed without merging**